docs/architecture/mbed-crypto-storage-specification.md

```diff
@@ -204,6 +204,7 @@ Integrated in Mbed OS TBD.
 
 * The layout of a key file now has a lifetime field before the type field.
 * Key files can store references to keys in a secure element. In such key files, the key material contains the slot number.
+* The type field has been split into a type and a bits field of 2 bytes each.
```

Comment:
The section you edited describes the format in Mbed TLS 2.22–2.24. Please leave this section alone and create a new section for the new format. This file deliberately preserves historical information.

Reply:
This section was specifically for version TBD. Should I add another section for version TBD underneath?

Reply:
Yes, please create a new section, also with version “TBD”.

We should fill in “TBD” at the latest we release a version of Mbed TLS, but that's a gap in our release process (or rather, the existing instance is a gap of the now-discontinued Mbed Crypto release process).